### PR TITLE
PLANNER-2435: Kogito task assigning phase1 blog post

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hi @ge0ffrey ,  while some style and formatting adjustments are sill needed I think It's in a reasonably shape for an initial review.

Would you mind have look?

Thanks.

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:9]: Can we make the first paragraph more answer the question "What is the benefit for me reading this?" What is the value proposition? Why should I care?

Referencing the previous blog is good, but I 'd leave that to the second paragraph, if you want them to read that first to better understand the problem. Readers are lazy, the TLDR of the problem/value should be in the first paragraph.

Readers might not now what Kogito is or what the benefit of kogito is.

Maybe something in the lines of
"In BPMN workflows in the cloud, we can automatically assign tasks to users by combining Kogito processes with OptaPlanner optimization. Furthermore, we can optimize those assignments, to ensure every user has the right skills for the tasks and ..."

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:119]:
```suggestion
Notice that the Spanish language was selected as the "Preferred language for communications", which establishes that:
```

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:111]: katy with a capital?

Writing docs guidelines is use active tense, which works well in blogs too:
"we can see that the task requires" => "The task requires ..."

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:129]: "is straightforward, you instruct" => "is straightforward: you instruct"

**Review by ge0ffrey — APPROVED** [review]

**Comment by ge0ffrey** [comment]
Looks good. Thanks for the blog, Walter!

**Comment by wmedvede** [comment]
Hi @ge0ffrey , your suggestions where applied, the blog post is ready.

Important notes:

1. before merging, we need to update the blog post date/directory to the real publication date. (not a big deal)
2. This blog post refers to the Task Assigning Service documentation that is under @hmanwani-rh review here https://github.com/kiegroup/kie-docs/pull/3689 at this moment. So we should ideally merge that PR before.

Thanks!

**Comment by ge0ffrey** [comment]
@yurloc this is good to merge, can you work with Walter to sync the merging with the date?

**Comment by wmedvede** [comment]
For the record, blog post date was moved to the 26th July as discussed. I'll keep an eye to make this merged (or changed if required) before this date.

**Pull request merged**
